### PR TITLE
Change frequency for all Xtensa chips

### PR DESCRIPTION
**Pull request by JurajSadel**

### CODE REVIEW

**Comment by bjoernQ** [comment]
We should probably update / change the FIXME comment since it's using CCOUNTER which is actually increased on every CPU clock cycle

@jessebraham 

Besides that I think we can for now use a fixed boot default - later when we do our first iteration on #44 we will get that value from there and even later that will be even changeable

File: esp-hal-common/src/delay.rs [hunk 1]
```diff
@@ -95,7 +95,7 @@ mod delay {
 
     // FIXME: The ESP32-S2 and ESP32-S3 have fixed crystal frequencies of 40MHz.
     //        This will not always be the case when using the ESP32.
```
Comment (jessebraham) [inline esp-hal-common/src/delay.rs:97]: Can you please either remove or update this comment (I had conflated crystal and clock frequencies when I made this anyways)?

Comment (jessebraham) [inline esp-hal-common/src/delay.rs:97]: Can you please either remove or update this comment (I had conflated crystal and clock frequencies when I made this anyways)?

Comment (JurajSadel) [reply esp-hal-common/src/delay.rs]: Yes, sure!

**Review by jessebraham — APPROVED** [review]
Thanks!

**Pull request merged**